keyboards/omron75/omron75.c

```diff
@@ -0,0 +1,31 @@
+#include "omron75.h"
+#include "led.h"
+
+void matrix_init_kb(void) {
+	// put your keyboard start-up code here
+	// runs once when the firmware starts up
+	matrix_init_user();
+	led_init_ports();
+};
+
+void matrix_scan_kb(void) {
+	// put your looping keyboard code here
+	// runs every cycle (a lot)
+	matrix_scan_user();
+};
+
+void led_init_ports(void) {
+    // * Set our LED pins as output
+    DDRB |= (1<<7);
+}
+
+void led_set_kb(uint8_t usb_led) {
```

Comment:
Can you use GPIO routines here? For example

```
  if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
    writePinLow(B7);
  } else {
    writePinHigh(B7);
  }
```

and in `matrix_init_kb` add in a 

```
setPinOutput(B7);
```